test/integration/etcd_storage_path_test.go

```diff
@@ -898,6 +521,78 @@ func TestEtcd3StoragePath(t *testing.T) {
 		t.Fatalf("error creating test namespace: %#v", err)
 	}
 
+	etcdStorageData := etcddata.GetEtcdStorageData()
+
+	// TODO storage is broken somehow.  failing on v1beta1 serialization
+	delete(etcdStorageData, gvr("admissionregistration.k8s.io", "v1alpha1", "initializerconfigurations"))
+
+	type hackOverride struct {
+		path string
+		gvk  *schema.GroupVersionKind
+	}
+
+	// TODO we store these in the wrong place and version in etcd
```

Comment:
Fix this, master team will be addressing those as part of the migration story we own: https://jira.coreos.com/browse/MSTR-213

Reply:
@deads2k I thought you fixed that one, didn't you?